static/src/javascripts-legacy/projects/common/modules/discussion/comment-count.js

```diff
@@ -54,7 +46,7 @@ define([
     }
 
     function getContentIds(indexedElements) {
-        return chain(indexedElements).and(keys).and(uniq).and(sortBy).join(',').value();
+        return Object.keys(indexedElements).sort().join(',');
```

Comment:
lodash keys would've returned an array composed of the key names from the indexedElements object, keys on an object should be unique anyway so I thought this was unnecessary?